dts/arm/st/stm32f302.dtsi

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2017 I-SENSE group of ICCS
```

Comment:
Please add your copyright

Reply:
fixed in 8719d57f

Reply:
Please fix it here.
You can do it like this: https://gist.github.com/bkemper/ca6ac68b174a047b5ccde3930c8568dc
or like this (my preferred option):
-Do the fix with commit message fixup! <sha_1 of the commit to fix>
-git rebase -i --autosquash HEAD~n (n > distance from fixup to commit to fix)